.github/workflows/linux.yml

```diff
@@ -9,6 +9,8 @@ on:
   pull_request:
     branches:
       - '*'
+  schedule:
+    - cron: "0 4 * * 0"
```

Comment:
If a scheduled job fails in over the weekend, will there be some notification? I've only ever configured Azure to have scheduled jobs + notifications...

If there are no notifications, I don't see much value in a scheduled job running as I doubt anyone would notice

Reply:
yes, there is an email notification depending on your settings (I get those, e.g., for mne-bids) -- and also the badges would show